Sources/Sentry/SentryRequest.h

```diff
@@ -0,0 +1,50 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(Request)
```

Comment:
Do I need to implement `hash`, `isEqual`, etc?

Reply:
Only if you have a specific case to use both function, otherwise we don't do it.